src/utils.rs

```diff
@@ -106,7 +106,47 @@ pub fn remove_dir_all_force(path: &Path) -> std::io::Result<()> {
             fs::remove_dir_all(path)
         }
         Err(e) => Err(e),
+    };
+
+    #[cfg(windows)]
+    {
+        if result.is_err() {
+            return try_remove_with_retry(path, None);
+        }
+    }
+
+    result
+}
+
+#[cfg(windows)]
+/// Retries clean up when encountered with OS 32 and OS 5 errors on Windows
+fn try_remove_with_retry(path: &Path, first_err: Option<std::io::Error>) -> std::io::Result<()> {
+    let max_retries = 5;
+    let mut attempts: i32 = if first_err.is_some() { 1 } else { 0 };
+    let mut last_err = first_err;
+
+    while attempts < max_retries {
+        if let Some(e) = &last_err {
+            tracing::debug!("Retrying deletion {}/{}: {}", attempts + 1, max_retries, e);
```

Comment:
This can be `info` I think so that the user knows what's happening and why things are slow.